build.sbt

```diff
@@ -77,18 +72,21 @@ def npmModulesTarballTask = Def.task {
   Seq(zipFile)
 }
 
-lazy val itServer = sys.env.get("PLAY_VERSION") match {
-  case Some("2.8") => itServerPlay28
-  case Some("2.9") => itServerPlay29
-  case _           => itServerPlay30
-}
+lazy val itServer = project
+  .settings(publish / skip := true)
+  .aggregate(
+    sys.env.get("PLAY_VERSION") match {
+      case Some("2.8") => itServerPlay28
+      case Some("2.9") => itServerPlay29
+      case _           => itServerPlay30
+    }
+  )
```

Comment:
this change was because in testing, I noticed that running
```
nvm install && PLAY_VERSION=2.8 sbt clean +test
```
was not running the test twice - once for 2.12 and once for 2.13 - it was just running once

so it's a bit more wordy but with this approach we can run the tests in all cross compiled scala versions for the current play version with
```
nvm install && PLAY_VERSION=2.8 sbt clean "project itServer" +test
```

going to add that to readme but I'll share with other devs first for feedback

Reply:
this will need an update to build-jobs before merging too